project/scripts/release.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Usage:
+# RELEASE_DIR=<directory to place fresh clones of projects> ./release.sh
```

Comment:
What it this `RELEASE_DIR`. Why not assume this is ran from within the dotty repo

Reply:
Because I wanted a fresh cloned copy with default remotes etc.